apps/hash-api/src/graph/knowledge/primitive/entity.ts

```diff
@@ -170,7 +171,9 @@ export const getEntities: ImpureGraphFunction<
   GetEntitiesRequest & { temporalClient?: TemporalClient },
   Promise<HashEntity[]>
 > = async ({ graphApi, temporalClient }, { actorId }, params) => {
-  await rewriteSemanticFilter(params.filter, temporalClient);
+  if (Predicate.hasProperty(params, "filter")) {
+    await rewriteSemanticFilter(params.filter, temporalClient);
+  }
```

Comment:
I think we need to move this functionality to the Graph or deal otherwise with semantic searches. Could you add a TODO with [this linear ticket](https://linear.app/hash/issue/BE-108/consider-moving-semantic-filter-rewriting-to-the-graph)?